### PR TITLE
Merge assignment 3 to assignment 4

### PR DESCRIPTION
**Pull request by cadencjk**

### CODE REVIEW

File: Auth/.gitignore
Comment (lhy-hoyin) [inline Auth/.gitignore:1]: Maybe keep the EOF new line?

File: Question/.gitignore
Comment (lhy-hoyin) [inline Question/.gitignore:1]: Keep EOF new line

File: User/.gitignore
Comment (lhy-hoyin) [inline User/.gitignore:1]: Keep EOF new line

File: User/initdb/02-host-privilege.sql
Comment (lhy-hoyin) [inline User/initdb/02-host-privilege.sql:1]: Keep EOF new line

File: frontend/src/pages/Question/QuestionDescription/QuestionDescription.css
Comment (lhy-hoyin) [inline frontend/src/pages/Question/QuestionDescription/QuestionDescription.css:1]: Add EOF new line

**Review by lhy-hoyin — CHANGES_REQUESTED** [review]
Generally okay, need some slight changes for consistency

File: Auth/.gitignore
Comment (cadencjk) [reply Auth/.gitignore:1]: Okay done

**Comment by cadencjk** [comment]
> Generally okay, need some slight changes for consistency

Okay added EOF line

**Review by lhy-hoyin — APPROVED** [review]
LGTM

**Pull request merged**